test/rules-google-event-measurement-fullstory.spec.ts

```diff
@@ -61,6 +61,20 @@ describe('Ruleset: Google Analytics Event Measurement to FullStory', () => {
         // now check that no calls get queued
         expect(await testHarness.popEvent(500)).to.be.undefined;
       });
+
+      it('gracefully handles a "clear" event', async () => {
+        await testHarness.execute(() => {
+          globalThis.dataLayer.push(
+            null,
+          );
+        });
+
+        const event = await testHarness.popEvent(500);
+        expectEqual(event, undefined);
+
+        const error = await testHarness.popError(500);
```

Comment:
Current status: This fails if running only Google Analytics ruleset tests and passes when running the entire test suite. Investigating if we have some global state carryover that's causing the difference.

Reply:
When running the full test suite the log appender had been changed away from `ConsoleAppender`. This could arguably be missed cleanup in other tests, but it also seemed reasonable that since that Node test harness is specifically inspecting `console` calls for errors that it ensures the `ConsoleAppender` is active -- so I've taken that approach.